src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml

```diff
@@ -105,100 +124,119 @@
+				<!-- Separator -->
+				<Rectangle Fill="{StaticResource MaterialOnSurfacePressedBrush}" 
+						   Height="1"
+						   HorizontalAlignment="Stretch" />
 
 				<!-- Privacy policy  -->
-				<Grid Margin="24,10,24,10">
+				<Grid Margin="24,10">
 					<Grid.ColumnDefinitions>
 						<ColumnDefinition Width="*" />
 						<ColumnDefinition Width="Auto" />
 					</Grid.ColumnDefinitions>
 
+					<!-- Label -->
 					<TextBlock Text="Privacy policy"
 							   x:Uid="Settings_PrivacyPolicy"
 							   VerticalAlignment="Center"
-							   Style="{StaticResource Subtitle2}"/>
+							   Style="{StaticResource Subtitle2}" />
 
-					<Button Grid.Column="1"
-							Command="{Binding NavigateToPrivacyPolicyPage}"
-							Style="{StaticResource MaterialButtonIconStyle}">
+					<!-- Button -->
+					<Button Command="{Binding NavigateToPrivacyPolicyPage}"
+							Style="{StaticResource MaterialButtonIconStyle}"
+							Grid.Column="1" >
 						<Button.Content>
 							<u:PathControl Style="{StaticResource RightArrowPathControlStyle}" />
 						</Button.Content>
 					</Button>
 				</Grid>
 
-				<Rectangle Height="1"
-						   HorizontalAlignment="Stretch"
-						   Fill="{StaticResource MaterialOnSurfacePressedBrush}" />
+				<!-- Separator -->
+				<Rectangle Fill="{StaticResource MaterialOnSurfacePressedBrush}" 
+						   Height="1"
+						   HorizontalAlignment="Stretch" />
 
 				<!-- Terms & conditions  -->
-				<Grid Margin="24,10,24,10">
+				<Grid Margin="24,10">
 					<Grid.ColumnDefinitions>
 						<ColumnDefinition Width="*" />
 						<ColumnDefinition Width="Auto" />
 					</Grid.ColumnDefinitions>
 
+					<!-- Label -->
 					<TextBlock Text="Terms &amp; conditions"
 							   x:Uid="Settings_TermsAndConditions"
-							   VerticalAlignment="Center"
-							   Style="{StaticResource Subtitle2}"/>
+							   Style="{StaticResource Subtitle2}"
+							   VerticalAlignment="Center" />
 
-					<Button Grid.Column="1"
-							Command="{Binding NavigateToTermsAndConditionsPage}"
-							Style="{StaticResource MaterialButtonIconStyle}">
+					<!-- Button -->
+					<Button Command="{Binding NavigateToTermsAndConditionsPage}"
+							Style="{StaticResource MaterialButtonIconStyle}"
+							Grid.Column="1" >
 						<Button.Content>
 							<u:PathControl Style="{StaticResource RightArrowPathControlStyle}" />
 						</Button.Content>
 					</Button>
 				</Grid>
 
-				<Rectangle Height="1"
-						   HorizontalAlignment="Stretch"
-						   Fill="{StaticResource MaterialOnSurfacePressedBrush}" />
+				<!-- Separator -->
+				<Rectangle Fill="{StaticResource MaterialOnSurfacePressedBrush}" 
+						   Height="1"
+						   HorizontalAlignment="Stretch"/>
 
 				<!-- Logout -->
 				<Button Content="LOGOUT"
 						x:Uid="Settings_Logout"
-						Margin="24,16"
-						HorizontalAlignment="Left"
 						Command="{Binding Logout}"
-						Background="{StaticResource MaterialErrorBrush}" />
+						Background="{StaticResource MaterialErrorBrush}"
```

Comment:
Logout is not an error. I don't think we should use an error brush for something that isn't an error.